### DISCUSSION TITLE
What is OCCPR?

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#998] [General]
Due to #994, what I see on https://orangec.readthedocs.io/en/latest/general/Occpr/ is only:

`The database has the file extension ".ods".`

I guess it's something like ctags. Please correct me if I'm wrong.

**Comment by LADSoft** [comment]
it is a database generator; the ide i had originally written had code completion and some other stuff and this was the program that generated the database for it.
